### ISSUE TITLE
PALGBDT implementation

### ISSUE THREAD

**Issue by kjappelbaum** [kjappelbaum/pyepal#65]
- Use quantile loss on a GBDT to predict the errorbars (using LightGBM as optional dependency 

We need to spend some thought on how to optimize the hyperparameters. I would expect GBDT to be much less sensitive to the hyperparameters than GPR but we should still update them at times. Cross validation is too expensive but I'm also not sure if we should include hyperopt or something like this as dependency ... 

Also the criterion is a bit less clear for GBDT as in the Bayesian case we can simply use the likely hood, here we would need some cross-validation
